test/06_locales.js

```diff
@@ -28,11 +28,53 @@ var moment = require('moment');
 var range = require('moment-range');
 var should = require('should');
 
-var Locales = require('../index');
+var Romcal = require('../index');
+var Utils = Romcal.Utils;
 
 describe('Testing localization functionality', function() {
 
   this.timeout(0);
 
+  // Locales extended with sample data
+  Romcal.Locales.frCA = Romcal.Locales.frCA || {};
+  Romcal.Locales.frCA.test = { hello: 'Allo', foo: 'foo' };
+  Romcal.Locales.en.test = { hello: 'Hello', foo: 'bar' };
+  Romcal.Locales.fr.test = { hello: 'Salut', lorem: 'ipsum' };
+
+  it('Should output French locale when locale is set to "fr"', function() {
```

Comment:
_Should_ → _This should_ unless by _Should_ is meant as the package/program name → then _output_ → _outputs_. I am not quite sure on this one though.

Same should apply on the similar cases below.